data/committee/sortition/sortition.go

```diff
@@ -33,15 +35,16 @@ func Select(money uint64, totalMoney uint64, expectedSize float64, vrfOutput cry
 	binomialN := float64(money)
 	binomialP := expectedSize / float64(totalMoney)
 
-	t := &big.Int{}
-	t.SetBytes(vrfOutput[:])
-
 	precision := uint(8 * (len(vrfOutput) + 1))
-	max, b, err := big.ParseFloat("0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", 0, precision, big.ToNearestEven)
+	maxFloatString := fmt.Sprintf("0x%s", strings.Repeat("f", crypto.DigestSize*2+1))
```

Comment:
This can't be right, can it?  Surely an odd number of hex digits is not what you want.

Reply:
You are correct. This was something I added while manually testing. Thanks!

Reply:
I'd prefer to see the `fmt.Sprintf` done once, rather than each time, as well.

Reply:
Added a local var since you can't assign the output of formatting to a constant.

Reply:
For that matter, perhaps you could move `max` out of the function as well, so it's not reparsed each time. I suspect the PR would be more palatable if it had a performance improvement, rather than only making the constant explicit. (On the explicitness front, why not `strings.Repeat("ff", crypto.DIgestSize)`?)

Reply:
Done!